auto_landing/src/tracking.cpp

```diff
@@ -0,0 +1,50 @@
+#include<auto_landing/tracking.hpp>
+
+
+namespace ariitk::auto_landing {
+
+ void Tracking::init(ros::NodeHandle& nh) {
+
+        set_firefly_pose_ = nh.advertise<geometry_msgs::PoseStamped>("/firefly/command/pose", 1);
+        husky_pose_ = nh.subscribe("/husky_velocity_controller/odom", 1, &Tracking::poseClbk,this);
```

Comment:
Try to give generic names  in subscribers and publishers, these can later be remapped to the correct topics in the launch file.